setup.py

```diff
@@ -36,7 +36,7 @@ def get_version(*file_paths):
     author='edX',
```

Comment:
Do you want to release a new version now, so that we try it out in edx-platform as soon a we're ready?